### PR TITLE
Add increment snippet

### PR DESCRIPTION
**Pull request by puf**
Add snippet for RTDB increment operator.

Upgrade to iOS 10, to allow using this operator.

Upgraded other dependencies.

Fix some tabs to spaces

### CODE REVIEW

File: database/DatabaseReference.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:DatabaseReference.xcodeproj">
```
Comment (morganchen12) [inline database/DatabaseReference.xcworkspace/contents.xcworkspacedata:5]: This should definitely not be committed

File: database/DatabaseReference/objc/ViewController.m [hunk 2]
```diff
@@ -161,4 +161,14 @@ - (void)writeNewUserWithCompletion:(FIRUser *)user withName:(NSString *)username
   // [END rtdb_write_new_user_completion]
 }
 
+- (void)incrementStarsForPost:(NSString *)postID byUser: (NSString *) userID {
+  // [START rtdb_post_stars_increment]
+  NSDictionary *updates = @{[NSString stringWithFormat: @"posts/%@/stars/%@", postID, userID]: @TRUE,
+                          [NSString stringWithFormat: @"posts/%@/starCount", postID]: [FIRServerValue increment: @1],
```
Comment (morganchen12) [inline database/DatabaseReference/objc/ViewController.m:167]: nit: no space after `:` in method parameter list (here and below)

```objc
[FIRServerValue increment:@1]
```

File: database/swift.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -158,6 +185,67 @@
...
+				"${TARGET_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/leveldb.framework",
+				"${TARGET_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/nanopb.framework",
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "\"${PODS_ROOT}/Target Support Files/Pods-DatabaseReference/Pods-DatabaseReference-frameworks.sh\"\n";
+			showEnvVarsInLog = 0;
+		};
+/* End PBXShellScriptBuildPhase section */
+
```
Comment (morganchen12) [inline database/swift.xcodeproj/project.pbxproj:248]: nit: You can get rid of this big project diff by running `pod deintegrate`.

**Review by morganchen12 — COMMENTED** [review]
The snippets themselves look good, had some mostly Xcode-related comments

File: database/DatabaseReference.xcworkspace/contents.xcworkspacedata (hunk 1 above)
Comment (puf) [reply database/DatabaseReference.xcworkspace/contents.xcworkspacedata:5]: I had no idea where it came from, so assumed it was an Xcode upgrade issue. Will revert.

File: database/swift.xcodeproj/project.pbxproj (hunk 3 above)
Comment (puf) [reply database/swift.xcodeproj/project.pbxproj:248]: Thanks for catching that, as I forgot to ask! Will do this tomorrow and update the PR.

File: database/DatabaseReference.xcworkspace/contents.xcworkspacedata (hunk 1 above)
Comment (puf) [reply database/DatabaseReference.xcworkspace/contents.xcworkspacedata]: This should be fixed now.

**Review by morganchen12 — APPROVED** [review]

**Pull request merged**